modules/admin_manual/pages/enterprise/external_storage/wnd_quick_guide.adoc

```diff
@@ -0,0 +1,155 @@
+** "--password-trim" removes blank characters from the password file added by 3rdparty software or other services.
++
+----
+[Unit]
+Description=ownCloud WND Listener for SERVER SHARE
+After=syslog.target
+After=network.target
+Requires=apache2.service
+[Service]
+User=www-data
+Group=www-data
+WorkingDirectory=/var/www/owncloud
+ExecStart=/usr/bin/php ./occ wnd:listen -vvv SERVER SHARE USER --password-file=/tmp/mypass --password-trim
+Type=simple
+StandardOutput=journal
+StandardError=journal
+SyslogIdentifier=%n
+KillMode=process
+RestartSec=1
+Restart=always
+[Install]
+WantedBy=multi-user.target
+----
+
+* Run the following command, once for each created file:
++
+[source,console]
+----
+sudo systemctl enable owncloud-wnd-listen-SERVER-SHARE.service
+sudo systemctl start  owncloud-wnd-listen-SERVER-SHARE.service
+----
+
+=== WND Process Queue Configuration
+
+Create or add a `crontab` file in `/etc/cron.d/oc-wnd-process-queue`.
+
+* Make a `crontab` entry to run a script iterating over all `SERVER SHARE` pairs with an appropriate `occ wnd:process-queue` command. 
+  The commands must be **strictly sequential**. 
+  This can be done by using `flock -n` and tuning the `-c` parameter of `occ wnd:process-queue`
+
+[source,console]
+----
+* * * * *  sudo -u www-data /usr/bin/php /var/www/owncloud/occ wnd:process-queue <HOST> <SHARE>
+----
+
+==== Execution Serialization
+
+Parallel runs of wnd:process-queue might lead to a user lockout. The reason for this is that several wnd:process-queue might use the same wrong password because it hasn’t been updated by the time they fetch it.
+
+It’s recommended to force the execution serialization of the wnd:process-queue command. You might want to use Anacron, which seems to have an option for this scenario, or wrap the command with flock.
+
+If you need to serialize the execution of the wnd:process-queue, check the following example with flock
+
+----
+* * * * * flock -n /tmp/wnd001 occ wnd:process-queue server1 share1
+* * * * * flock -n /tmp/wnd002 occ wnd:process-queue server1 share2
+* * * * * flock -n /tmp/wnd003 occ wnd:process-queue server2 share3
+----
+
+== Troubleshooting
+
+* process queue will not work if there is a backslash in the share path configured in webui
+
+If you encounter issues using Windows network drive, then try the following troubleshooting steps:
+
+First check the connection to the share by using smbclient on the command line of the ownCloud server. Here is an example:
+
+----
+smbclient -U Username -L //Servername
+----
+
+Take the example of attempting to connect to the share named MyData using occ wnd:listen. Running the following command would work:
+
+----
+sudo -u www-data php occ wnd:listen MyHost MyData svc_owncloud password
+----
+
+The command is case sensitive, and that it must match the information from the mount point configuration.
```

Comment:
LGTM, just one more addition at the very end in the trouble shooting section:
- if the output of the `occ process-queue ..` command shows `0 Storages found` this means that there was no corresponding external storage configuration found, for example because the casing between calling the process queue and the web interface does not exactly match.